bin/yb-ctl

```
@@ -977,20 +1025,22 @@ class ClusterControl:
         cmd = [yb_admin_binary_path, "--master_addresses", self.options.master_addresses,
                "change_master_config", cmd_type, daemon_id.get_ip_address(),
                str(self.options.base_ports[DAEMON_TYPE_MASTER]["rpc"])]
-        wait_count = 0
-        while wait_count < MAX_WAIT_ITERS:
+
+        def call_change_config():
             try:
-                subprocess.check_call(cmd)
-                return
+                with open(os.devnull, "wb") as devnull:
+                    subprocess.check_output(cmd, stderr=devnull)
+                return True
             except subprocess.CalledProcessError:
-                wait_count += 1
-                time.sleep(SLEEP_TIME_IN_SEC)
+                pass
```

See my earlier comment regarding silently ignoring errors.